lib/btoa.js

```diff
@@ -43,22 +43,14 @@ function btoa(s) {
  * Lookup table for btoa(), which converts a six-bit number into the
  * corresponding ASCII character.
  */
-function btoaLookup(idx) {
-  if (idx < 26) {
-    return String.fromCharCode(idx + "A".charCodeAt(0));
-  }
-  if (idx < 52) {
-    return String.fromCharCode(idx - 26 + "a".charCodeAt(0));
-  }
-  if (idx < 62) {
-    return String.fromCharCode(idx - 52 + "0".charCodeAt(0));
-  }
-  if (idx === 62) {
-    return "+";
-  }
-  if (idx === 63) {
-    return "/";
+const keystr =
+  "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/=";
```

Comment:
Does the `'='` on the end of this string ever get used? It looks like in both cases the maximum possible index is 63? (and `keystr[64] == '='`)

Reply:
You are right, I copied this straight away from the implementation linked on the first comment and never thought twice about the length of the string. 
Tests are all green even when removing the string, doing that now!